x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -64,7 +64,13 @@ function ensureGeometryType(type, expectedTypes) {
  * @param {string} geoFieldType Geometry field type ["geo_point", "geo_shape"]
  * @returns {number}
  */
-export function hitsToGeoJson(hits, flattenHit, geoFieldName, geoFieldType) {
+export function hitsToGeoJson(
+  hits,
+  flattenHit,
+  geoFieldName,
+  geoFieldType,
+  epochMillisFields = []
```

Comment:
I'd just make this a required param. this isn't public API that benefits from sugar. It's only called here https://github.com/elastic/kibana/blob/045122f89f3a77721c17c9f2d52444ba4e3da25d/x-pack/plugins/maps/public/layers/sources/es_search_source/es_search_source.js#L398 and actually passed in anyway.